### ISSUE TITLE
Organizr-v2 defaults to setting page instead of homepage

### ISSUE THREAD

**Issue by brando56894** [causefx/Organizr#1110]
###### Operating System: Linux
<hr>

##### Problem Description:

Not sure if this is a misconfiguration on my end or a bug, but whenever I login to Organizr it brings me to the settings page instead of the Homepage which I have enabled. I see no options to select which page to display at login, unless I'm missing something...
<hr>

##### Reproduction Steps:

Just login. This has been happening since the beta.
<hr>

#### Errors on screen?  If so paste here:

```
N/A
```

**Comment by causefx** [comment]
screenshot your tabs page so I may see please

**Comment by Roxedus** [comment]
There is a default setting. https://img.roxedus.net/chrome-2019-03-13_22-59-23.png

**Comment by brando56894** [comment]
Damn you guys are quick! Also thanks @si0972 that did it, How did I not notice this when setting up my tabs? Now I feel dumb hahaha

One last question, is there a way to make the calendar only show the current week by default instead of the whole month?

**Comment by Roxedus** [comment]
Yes. https://img.roxedus.net/chrome-2019-03-13_23-03-41.png

**Comment by brando56894** [comment]
Yep, just saw it while waiting for an answer. I must be blind! 

https://i.ytimg.com/vi/BNsrK6P9QvI/hqdefault.jpg

**Comment by teotikalki** [comment]
Having the same problem. Solutions which are links to pictures that are no longer available are... bad. Can the solution please be posted in a way that is still viewable a few months later?

**Comment by Roxedus** [comment]
@teotikalki Here
![image](https://user-images.githubusercontent.com/7110194/69907706-05d70f80-13da-11ea-8da1-7f230cb83094.png)

**Comment by teotikalki** [comment]
@Roxedus Thanks. I guess I was hoping to see the popup part? That part I had set. 
Related question maybe: How do I get the house icon? It's not available when I make the homepage tab.

I had read that the tab url should be 'homepage.php'. Is that correct? Is there a local url? What you're showing looks to be part of what's currently giving me a blank page.
-> Question: SHOULD anything be on this page by default? I don't use Plex or any Mono (cr)App, so the tutorials I've found haven't been relevant. I'd love to see something like KODI recently added, but for proof-of-concept I was just expecting to see 'home' on a page or something.
